### PR TITLE
GCE: Update Google API dependencies

### PR DESCRIPTION
**Pull request by nicksardo**
Updating `google.golang.org/api/` 

**Special notes for your reviewer**:
/assign @bowei

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by bowei** [comment]
seems like a compile error in run_remote.go?
```
W0817 16:53:44.110] test/e2e_node/runner/remote/run_remote.go:372: cannot use val (type string) as type *string in field value
W0817 16:53:44.110] test/e2e_node/runner/remote/run_remote.go:546: cannot use true (type bool) as type *bool in field value
W0817 16:53:44.110] test/e2e_node/runner/remote/run_remote.go:630: cannot use item.Value (type *string) as type string in argument to strings.HasPrefix
```

**Comment by nicksardo** [comment]
Yep, the v0.beta library changed to use pointers. 
/assign @vishh

**Review by vishh — COMMENTED** [review]
LGTM

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -430,32 +430,32 @@
 		},
 		{
 			"ImportPath": "github.com/containernetworking/cni/libcni",
-			"Comment": "v0.5.2",
+			"Comment": "spec-v0.3.1",
```
Comment (bowei) [inline Godeps/Godeps.json]: how come the version seems to have moved backwards

Comment (bowei) [inline Godeps/Godeps.json]: resolved

**Pull request merged**